controllers/securitypolicy_controller.go

```diff
@@ -471,6 +490,11 @@ func (r *SecurityPolicyReconciler) updateStatusOfLinkedAPIs(ctx context.Context,
 	}
 
 	for _, a := range policy.Spec.AccessRightsArray {
+		ok := oldLinks[types.NamespacedName{Name: a.Name, Namespace: a.Namespace}.String()]
```

Comment:
can you please elaborate this part? I am a bit confused

Reply:
oldLinks- refers to APIs that already had a link to policy.(Linked in previous reconcile)

The loop goes through all the APIs that are linked currently(policy.Spec.AccessRightsArray ).
It adds links to the APIs, ignoring APIs that already have a link(oldLinks) to avoid unnecessary reconciliations.

I hope it's clear now.

Reply:
ah, makes super sense! thanks for explanation